signature-specification.md

```diff
@@ -0,0 +1,230 @@
+# Signature Specification
+
+This document describes how Notary v2 signatures are created and stored.
+The document consists of following sections:
+* **[Storage](#storage)**: Describes how signatures are stored in OCI registry
+* **[Signature Envelope](#signature-envelope)**: Describes how signatures are created.
+
+
+## Storage
+This section describes how Notary v2 signatures are stored in the OCI Distribution conformant registry.
+Notary v2 uses [ORAS artifact manifest](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md) to store the signature in the repository. The signature artifact manifest consists of a signature type, a reference to the manifest of the artifact being signed, a blob referencing to the signature, and a collection of annotations.
+
+-<img src="media/signature-specification.jpg" width="550" height="450">
```

Comment:
Replace the HTML tag with a Markdown tag for the image.

Reply:
Github won't recognize the syntax of [](.. =400) or [](... =400px), so to resize the image I've to use the `<img/>` syntax.